lib/serializer.js

```diff
@@ -154,6 +155,10 @@ function _getSerializedContext(err) {
         _.omit(err, self._knownFields) :
         {};
 
+    if (topLevelFields.domain instanceof domain.Domain) {
+        topLevelFields = _.omit(topLevelFields, [ 'domain' ]);
```

Comment:
When using domains, there will also be a boolean field added by node called `domainThrown`.  Since it's a small boolean, I figure there's no harm in keeping it in the serializer.  Let me know if anyone feels differently and I can add that to the omission list.

Reply:
I agree with @cprussin: keeping the `domainThrown` field seems at worst harmless, so I'm good with this.